lib/krane/deploy_task.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'kubernetes-deploy/deploy_task'
+
+module Krane
+  class DeployTask < KubernetesDeploy::DeployTask
+    def initialize(**args)
```

Comment:
Not sure how much this matters, but technically doesn't this mean `Krane::DeployTask` _does_ have the `allow_globals` argument (you won't get an argument error), but we overwrite it? We could manually raise the argument error I guess.